File: MultiLangTest/results/MultiLangTest.py
class Calculator:
    def factor(self, n):
        if n <= 1:
            return 1
        else:
            return self.factor(n - 1) * n

print "Hello!"

arr = [1, 2, 3]
arr.append(4)

print "n = %s, arr[0] = %s" % (len(arr), arr[0])

map = {
  "a": 2,
  "b": 3,
}
print "map['a'] = %s, arr[1] = %s" % (map["a"], arr[1])

if arr[0] == 1:
    print "TRUE-X"
else:
    print "FALSE"

sum = 0
i = 0
while i < 10:
    sum += i + 2
    i += 1
print sum

calc = Calculator()
print "5! = %s" % (calc.factor(5))

File: MultiLangTest/results/MultiLangTest.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.HashMap;
import java.util.List;

class Calculator {
    public Integer factor(Integer n) throws Exception
    {
        if (n <= 1) {
            return 1;
        } else {
            return this.factor(n - 1) * n;
        }
    }
}

class Program {
    public static void main(String[] args) throws Exception {
        System.out.println("Hello!");
        
        List<Integer> arr = new ArrayList<Integer>(Arrays.asList(1, 2, 3));
        arr.add(4);
        
        System.out.println("n = " + arr.size() + ", arr[0] = " + arr.get(0));
        
        HashMap<String, Integer> map = new HashMap<String, Integer>();
        map.put("a", 2);
        map.put("b", 3);
        System.out.println("map['a'] = " + map.get("a") + ", arr[1] = " + arr.get(1));
        
        if (arr.get(0) == 1) {
            System.out.println("TRUE-X");
        } else {
            System.out.println("FALSE");
        }
        
        Integer sum = 0;
        for (Integer i = 0; i < 10; i++) {
            sum += i + 2;
        }
        System.out.println(sum);
        
        Calculator calc = new Calculator();
        System.out.println("5! = " + calc.factor(5));
    }
}